packages/dashboard/src/components/CompositeList/CompositeList.styles.ts

```diff
@@ -88,17 +88,17 @@ export const getCommandBarSelectionCountItemStyles = (): Partial<IButtonStyles>
   };
 };
 
-export const getDetailsListStyle = (): IStyle => {
+export const getDetailsListStyle = (showCommandBar: boolean): IStyle => {
```

Comment:
Should this be optional for backwards compatibility?

Reply:
I have updated it to be an optional prop.